unittests/test_utility.py

```diff
@@ -29,6 +29,13 @@ def test_command_success():
     assert completed.stdout == 'foobar\n'
 
 
+def test_command_stdin_isatty():
```

Comment:
This unit test does not fail if you run it with the current master.

Reply:
In principle, the test is correct, but it somehow interferes with the stdout capturing of pytest. If you run `-s`, it fails as expected, but not if you let pytest capture the output.

Reply:
I don't think that we can reliably reproduce this from within the unit tests, because pytest always sets stdin to null, unless you pass `-s`. I will remove the unit test then.